cli/lsp/tsc.rs

```diff
@@ -94,19 +93,28 @@ use tower_lsp::jsonrpc::Error as LspError;
 use tower_lsp::jsonrpc::Result as LspResult;
 use tower_lsp::lsp_types as lsp;
 
-static BRACKET_ACCESSOR_RE: Lazy<Regex> =
-  lazy_regex!(r#"^\[['"](.+)[\['"]\]$"#);
```

Comment:
> Why this change? `lazy_regex` evaluates the regex at compile time from my understanding and now we're no longer doing that.

Because it has this error.  Correct me if I am wrong.

```bash
error[E0308]: mismatched types
  --> cli\lsp\tsc.rs:98:20
   |
98 |   LazyLock::new(|| lazy_regex!(r#"^\[['"](.+)[\['"]\]$"#));
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `Regex`, found `Lazy<_, ...>`
   |
   = note: expected struct `regex::Regex`
              found struct `lazy_regex::Lazy<_, {closure@cli\lsp\tsc.rs:98:20: 98:58}>`
   = note: this error originates in the macro `lazy_regex` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0308`.
error: could not compile `deno` (bin "deno") due to 1 previous error
```

Reply:
I think it can be changed to:

```rs
static BRACKET_ACCESSOR_RE: lazy_regex::Lazy<Regex> =
  lazy_regex!(r#"^\[['"](.+)[\['"]\]$"#);
```